### PR TITLE
Userspace: TaskBar with scroll

### PR DESCRIPTION
**Pull request by nimelehin**
TaskBar

### CODE REVIEW

**Comment by awesomekling** [comment]
I don't think this is a good UI. I would much prefer the classical thing where the buttons get smaller so that more of them can fit.

**Comment by zlotny** [comment]
Agree with being a little off. If for some reason you have 100 things opened the stack would be so large that finding a window scrolling would be really annoying. 

However the StackView implementation is neat :) How about keeping that? Maybe put an example of the StackView implementation on WidgetGallery?

File: Libraries/LibGUI/AbstractStackView.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```
Comment (zlotny) [inline Libraries/LibGUI/AbstractStackView.cpp:2]: Also, those copyright headers show wrong information.

**Comment by Pulgovisk** [comment]
I think the concept of rows would be better
image

**Comment by awesomekling** [comment]
Closing this for lack of activity. Feel free to reopen if/when progress is made :)

**Comment by nimelehin** [comment]
I'm sorry for the delays - exams(. @awesomekling what do you think about StackView. Is it a thing to keep?

**Pull request closed without merging**